geometry_msgs/Point[] keyframe_pts
geometry_msgs/Point[] motion_pts
